src/Integration/LocalServices/ProjectSystemHelper.cs

```diff
@@ -24,6 +24,12 @@ internal class ProjectSystemHelper : IProjectSystemHelper
         internal const string TestProjectKind = "{3AC096D0-A1C2-E12C-1390-A8335801FDAB}";
         internal static readonly Guid TestProjectKindGuid = new Guid(TestProjectKind);
 
+        /// <summary>
+        /// This is the HResult returned by IVsBuildPropertyStorage when attempting to
+        /// read a property that does not exist.
+        /// </summary>
+        private const int E_XML_ATTRIBUTE_NOT_FOUND = unchecked((int)0x8004C738);
```

Comment:
No public definition of this error code. This is returned when we query for a project property which does not exist. This is OK and should not be treated as a failure, so we need to check for this.